client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -189,6 +215,51 @@ const WebServerSettingsCard = ( {
 		);
 	};
 
+	const getWpVersionContent = () => {
+		if ( isGettingWpVersion ) {
+			return;
+		}
+
+		const isWpVersionButtonDisabled =
+			disabled || ! selectedWpVersion || selectedWpVersion === wpVersion;
+		const selectedWpVersionValue =
+			selectedWpVersion || wpVersion || ( disabled && recommendedValue );
+
+		return (
+			<FormFieldset>
+				<FormLabel>{ translate( 'WordPress version' ) }</FormLabel>
+				<FormSelect
+					disabled={ disabled || isUpdatingPhpVersion }
+					className="web-server-settings-card__wp-version-select"
+					onChange={ ( event ) => setSelectedWpVersion( event.target.value ) }
+					value={ selectedWpVersionValue }
+				>
+					{ getWpVersions().map( ( option ) => {
+						return (
+							<option
+								disabled={ option.value === phpVersion }
```

Comment:
```suggestion
								disabled={ option.value === wpVersion }
```

Although is disabling the currently selected version necessary? It means switch from Latest to Beta you can't change your mind and go back to Latest again, which means you can't re-hide the button.

Reply:
I think the behavior is odd too, but it's how the select works for the other fields. We can reconsider with https://github.com/Automattic/dotcom-forge/issues/2892